### ISSUE TITLE
Colours

### ISSUE THREAD

**Issue by cathydutton** [DEFRA/design-system-backlog#5]
### What
An extension of the GOV.UK colour palette.

### Why
To maintain consistency across Defra's internal services

**Comment by cathydutton** [comment]
![brand](https://user-images.githubusercontent.com/5614324/58030272-2c0c2080-7b16-11e9-91fc-0b785f721923.jpg)

**Comment by cathydutton** [comment]
![links](https://user-images.githubusercontent.com/5614324/58029905-78a32c00-7b15-11e9-97a7-9c6c4cbfd7c0.jpg)

**Comment by benwakefield** [comment]
Something we found whilst testing is that the current gov design template colour ratio / tones could be improved to help users with perceptual conditions such as [Irlens syndrome](https://irlen.com/what-is-irlen-syndrome/). It is reported to affect 12-15% of the UK population and commonly causes symptoms such as headaches, dizziness, tiredness

The colour bar at the top of the [Irlens website](https://irlen.com/what-is-irlen-syndrome/) allows users to adjust the background tone to create a more perceivable web page. A feature like this (or simply reducing the current high contrast ratio) could benefit internal users and improve accessibility across GOV if the issue is upstreamed

**Comment by glancyea** [comment]
@cathydutton 
1. I'd argue that familiarity/consistency are more important than brand for links - so why not standard blue?
2. I think that visited link colour being different to the standard link colour is a helpful marker for those who can see it.

**Comment by cathydutton** [comment]
Good point on the colours @glancyea. I took them from here https://www.gov.uk/government/organisations/department-for-environment-food-rural-affairs but I'm not sure if we have to use them or not.

**Comment by benwakefield** [comment]
I had confirmation from Design102 (the agency responsible for the group HM Gov rebrand) that DEFRA green should be #00a33b

**Comment by cathydutton** [comment]
Maybe this is where the other colour came from?

<img width="506" alt="Screen Shot 2019-07-18 at 09 01 56" src="https://user-images.githubusercontent.com/5614324/61440946-95ed4e80-a93c-11e9-80c7-2c0f27bab1fb.png">

**Comment by benwakefield** [comment]
Must be a legacy branding thing, APHA isn't listed in the organisation.scss and if you check on GOV https://www.gov.uk/government/organisations/animal-and-plant-health-agency they've aligned it with #00a33b

**Comment by cathydutton** [comment]
rgb(0, 163, 59);
#00a33b
hsl(142, 100%, 32%)
